lib/templates.ts

```diff
@@ -22,6 +22,8 @@ import { isValidISODate } from "./utils/date/isValidISODate";
 import { validateTemplate } from "@lib/utils/form-builder/validate";
 import { dateHasPast } from "@lib/utils";
 import { validateTemplateSize } from "@lib/utils/validateTemplateSize";
+import { NotificationsInterval } from "packages/types/src/form-types";
+import { validateNotificationsInterval } from "app/(gcforms)/[locale]/(form filler)/id/[...props]/lib/notifications";
```

Comment:
should the notifications.ts file perhaps live in this `/lib` directory instead of with the page code?  This could avoid any potential issues where the code is included/called by accident directly from client side code.

Reply:
I was just coding exactly that :)